### ISSUE TITLE
Browserify Error

### ISSUE THREAD

**Issue by infacq** [angular/material#3280]
here is my code before run `gulp` to browserify it all

````js
var angular = require('angular');
angular.module('jobApp', [require('angular-material')]).controller('MainCtrl', ['$scope', function ($scope) {
	'use strict';
	// Initialize the scope variables
	$scope.$watch('appname', function () {
		console.log($scope.file, $scope.appname);
	});
}]);
````

when run in browser, this error will come out

````bash
TypeError: Cannot read property '' of undefined
    at lookup (bundle.js:632)
    at animationRunner (bundle.js:696)
    at performAnimation (bundle.js:1299)
    at bundle.js:1232
    at bundle.js:554
    at Scope.$get.Scope.$digest (bundle.js:33571)
    at Scope.$get.Scope.$apply (bundle.js:33789)
    at bootstrapApply (bundle.js:19593)
    at Object.invoke (bundle.js:22391)
    at doBootstrap (bundle.js:19591)
````

at line 632 I just found this

````js
if ($sniffer.transitions || $sniffer.animations) {
    matches.push($injector.get(selectors['']));
}
````

on another line 30183
````js
    function consoleLog(type) {
      var console = $window.console || {},
          logFn = console[type] || console.log || noop,
          hasApply = false;

      // Note: reading logFn.apply throws an error in IE11 in IE8 document mode.
      // The reason behind this is that console.log has type "object" in IE8...
      try {
        hasApply = !!logFn.apply;
      } catch (e) {}

      if (hasApply) {
        return function() {
          var args = [];
          forEach(arguments, function(arg) {
            args.push(formatError(arg));
          });
          return logFn.apply(console, args);
        };
      }
````
does anyone familiar with this error?

**Comment by ThomasBurleson** [comment]
This appears to be an Angular 1.4.x or a Browserify issue. Please provide a CodePen or Plunkr that demonstrates this issue. Here are some starter demo templates that you can use/fork:

*  CodePen template: http://codepen.io/team/AngularMaterial/pen/BNNqWm
*  Plunkr template: http://plnkr.co/edit/yP0gB9iTacIQb48thud8?p=preview.

We will reopen if you provide a demo.

**Comment by gkalpak** [comment]
@infacq, could you check if your issue is resolved by 7ca139a (the stacktrace is similar to another issue) ?

**Comment by infacq** [comment]
@ThomasBurleson your template for codepen is 404. as for Plunkr, the template didn't provide for browserify. 

@gkalpak I am not sure.

BTW, I have delete the node_modules folder and reinstall again, all of sudden no more error as before.
